res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -47,7 +47,6 @@ limitations under the License.
         .mx_BaseCard_close {
             margin-inline-end: calc(var(--BaseCard_header_button-margin) - 4px);
         }
-    }
 
     .mx_ThreadPanel__header {
```

Comment:
Despite being confusing a little bit, `.mx_ThreadPanel__header` is a child element of `.mx_BaseCard_header`.